Extension/package.json

```diff
@@ -280,6 +280,19 @@
           "severity": 4,
           "message": 5
         }
+      },
+      {
+        "name": "iar",
+        "source": "iar",
+        "owner": "cpptools",
+        "fileLocation" : "absolute",
+        "pattern": {
+          "regexp": "^\"(.*?)\",(\\d+)\\s+(?:[Ff]atal\\s+)?([Ww]arning|[Ee]rror)(\\[\\w+\\d+]:\\s+.*)$",
```

Comment:
`(\\[\\w+\\d+]:\\s+.*)` can be `\\[(\\w+\\d+)\\]:\\s(+.*)` if you use

```json
"code": 4,
"message": 5
```

to enable displaying the error code separately from the message.

Reply:
Done